csharp/tests/Facebook.Yoga.Desktop.Tests/Facebook.Yoga.Desktop.Tests.csproj

```diff
@@ -0,0 +1,103 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Allows the desktop tests to be run on windows.... The generated tests target NUnit 2.6.4 requiring this project to utilize that Nuget version...